pkg/prometheus/promcfg_test.go

```diff
@@ -2553,10 +2569,6 @@ func TestEnforcedNamespaceLabelOnExcludedPodMonitor(t *testing.T) {
 						"group": "group1",
 					},
 				},
-				TypeMeta: metav1.TypeMeta{
-					APIVersion: monitoring.GroupName + "/" + monitoringv1.Version,
-					Kind:       monitoringv1.PodMonitorsKind,
-				},
```

Comment:
Is that intentional? 🤔

Reply:
There are other occurrences in the same file

Reply:
Ups!
It was by mistake, I fixed, can you check pls?